From: Marc Deslauriers <marc.deslauriers@canonical.com>
Date: Fri, 19 Aug 2011 11:50:11 -0400
Subject: virt-manager: Fix "Resize to VM" menu option

Hello,

This patch fixes the "Resize to VM" menu option by correcting inverted return
values in autodrawer.py.

Thanks,

Marc.

commit 8e666f4b87926c866ed35e58eb82213d2c514e89
Author: Marc Deslauriers <marc.deslauriers@ubuntu.com>
Date:   Fri Aug 19 11:46:48 2011 -0400

    Fix inverted width and height

Origin: upstream, http://git.fedorahosted.org/git/?p=virt-manager.git;a=commitdiff;h=a815fa17a7c643ddde8ea326399e2bee449dff98
---
 src/virtManager/autodrawer.py |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/virtManager/autodrawer.py b/src/virtManager/autodrawer.py
index 56a9a80..b82b47b 100644
--- a/src/virtManager/autodrawer.py
+++ b/src/virtManager/autodrawer.py
@@ -256,10 +256,10 @@ class OverBox(parentclass):
         self.underWin = None
 
     def do_size_request(self, req):
-        height, width = self._size_request()
+        width, height = self._size_request()
 
-        req.height = height
         req.width = width
+        req.height = height
 
     def do_size_allocate(self, newalloc):
         self.allocation = newalloc
-- 
